File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let uniqueIndex = 0;

    for(let i = 1; i < nums.length; i++) {
        if(nums[i] !== nums[uniqueIndex]) {
            uniqueIndex ++;
            nums[uniqueIndex] = nums[i] ;
        }
    }
    
    return uniqueIndex +1;
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        unique_index = 0
        for i in range(1,len(nums)):
            if nums[i] != nums[unique_index]:
                unique_index += 1
                nums[unique_index] = nums[i]
        
        return unique_index + 1

    